jsx/src/components/Groups/Groups.jsx

```diff
@@ -74,11 +71,12 @@ const Groups = (props) => {
                 )}
               </ul>
               <PaginationFooter
-                endpoint="/groups"
-                page={page}
+                offset={offset}
                 limit={limit}
-                numOffset={slice[0]}
-                numElements={groups_data.length}
+                visible={groups_data.length}
+                total={total}
+                next={() => setOffset(offset + limit)}
+                prev={() => setOffset(offset - limit)}
```

Comment:
It's (technically) valid to have `offset=5, limit=10`, in which this should round to 0.

```suggestion
                prev={() => setOffset(offset >= limit ? offset - limit : 0 )}
```
```